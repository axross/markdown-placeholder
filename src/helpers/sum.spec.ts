import { describe, expect, it } from "vitest";
import { sum } from "./sum";

describe("sum()", () => {
  it("returns the sum of the given two values", () => {
    // eslint-disable-next-line no-magic-numbers
    expect(sum(2, 5)).toBe(7);
  });

  it("matches with the result snapshot", () => {
    // eslint-disable-next-line no-magic-numbers
    expect(sum(3, 8)).toMatchSnapshot();
  });
});
