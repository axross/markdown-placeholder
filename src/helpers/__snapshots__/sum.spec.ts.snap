// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`sum() > matches with the result snapshot 1`] = `11`;
